src/main/java/org/openstreetmap/atlas/proto/adapters/ProtoPrimitiveLongArrayAdapter.java

```diff
@@ -0,0 +1,53 @@
+package org.openstreetmap.atlas.proto.adapters;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.openstreetmap.atlas.proto.ProtoPrimitiveLongArray;
+import org.openstreetmap.atlas.utilities.arrays.LongArray;
+import org.openstreetmap.atlas.utilities.conversion.LongArrayToListOfByteArrayConverter;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.protobuf.ByteString;
+
+/**
+ * Provides an interface for client code to obtain a ready_to_serialize LongArray in the form of a
+ * ProtoPrimitiveLongArray. Also provides a method to obtain a LongArray back from a
+ * ProtoPrimitiveLongArray.
+ *
+ * @author lcram
+ */
+public class ProtoPrimitiveLongArrayAdapter
+{
+    /*
+     * TODO perhaps this class would be better implemented as a TwoWayConverter and not an Adapter?
+     */
```

Comment:
This class initially was an Adapter class, but I ended up changing it to more of a Converter model. This might need a change.